src/AppInstallerCLITests/TestData/ManifestV1-MultiFile-DefaultLocale.yaml

```diff
@@ -0,0 +1,23 @@
+PackageIdentifier: microsoft.msixsdk
```

Comment:
These split manifests should be in a directory for organization and testing.  Was it complicating the copy on build?